docker-compose.dev.yml

```diff
@@ -124,6 +124,7 @@ services:
       ALEPH_PROFILE: "false"
       ALEPH_SECRET_KEY: "development"
       FLASK_APP: "aleph.wsgi"
+      FLASK_ENV: "development"
```

Comment:
The fix also works without `FLASK_ENV` here. Also, `FLASK_ENV` [is deprecated](https://github.com/pallets/flask/issues/4714)

Reply:
Done